test/libsolidity/util/SoltestTypes.h

```diff
@@ -229,6 +230,32 @@ struct FunctionCallArgs
 	}
 };
 
+/// Units that can be used to express function value
+enum class FunctionValueUnit
+{
+	Wei,
+	Ether
+};
+
+/// FunctionValue structure
+/// Holds value along with coin it is expressed in
```

Comment:
```suggestion
/// Holds value along with unit it is expressed in
```
Please add: After parsing, the value is always converted to the base unit, even though `unit` stays unchanged.

Reply:
I suggest that I add another field to the structure which will be valueInOriginalUnit instead of . What do you think?

Reply:
I think it's fine as it is, it just needs to be documented.